contracts/l1-l2-messaging/run_e2e.sh

```diff
@@ -60,13 +65,14 @@ curl -H 'Content-Type: application/json' \
      http://127.0.0.1:5050/postman/send_message_to_l2
 
 # Send back some balance to consume manually.
-starkli invoke 0x03c80468c8fe2fd36fadf1b484136b4cd8a372f789e8aebcc6671e00101290a4 withdraw 0x1 0x2 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+echo "INVOKE"
+starkli invoke $CONTRACT_L2 withdraw 0x1 0x2 $CONTRACT_L1 --max-fee-raw $MAX_FEE
 
 curl -H 'Content-Type: application/json' \
      -d '{
-          "from_address": "0x34ba56f92265f0868c57d3fe72ecab144fc96f97954bbbc4252cef8e8a979ba",
-          "to_address": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512", "payload": ["0x0","0x1","0x2"]
+          "from_address": "0x03c80468c8fe2fd36fadf1b484136b4cd8a372f789e8aebcc6671e00101290a4",
```

Comment:
can the constants be embedded here as well instead?

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/299/commits/1c332514eb30341dde6eb9fe743d07450c6ba77c